docs/README.md

```diff
@@ -32,23 +32,23 @@ downloads and installs the distribution.
 It has to be invoked only once.
 It uses environment variables as parameters:
 
-| Parameter               | Description                                                      | Required | Default value             |
-|-------------------------|------------------------------------------------------------------|----------|---------------------------|
+| Parameter               | Description                                                      | Required | Default value               |
+|-------------------------|------------------------------------------------------------------|----------|-----------------------------|
 | `OTEL_DOTNET_AUTO_HOME` | Location where binaries are to be installed                      | No       | `$HOME/.splunk-otel-dotnet` |
-| `OS_TYPE`               | Possible values: `linux-glibc`, `linux-musl`, `macos`, `windows` | No       | *Calculated*              |
-| `TMPDIR`                | Temporary directory used when downloading the files              | No       | `$(mktemp -d)`            |
-| `VERSION`               | Version to download                                              | No       | `v0.0.1-alpha.2`           |
+| `OS_TYPE`               | Possible values: `linux-glibc`, `linux-musl`, `macos`, `windows` | No       | *Calculated*                |
+| `TMPDIR`                | Temporary directory used when downloading the files              | No       | `$(mktemp -d)`              |
+| `VERSION`               | Version to download                                              | No       | `v0.1.0-beta.1`             |
 
 The [instrument.sh](../instrument.sh) script
 enables the instrumentation in the current shell session.
 It has to be run before you run a .NET application.
 uses environment variables as parameters:
 
-| Parameter               | Description                                                            | Required | Default value             |
-|-------------------------|------------------------------------------------------------------------|----------|---------------------------|
-| `ENABLE_PROFILING`      | Whether to set the .NET CLR Profiler, possible values: `true`, `false` | No       | `true`                    |
+| Parameter               | Description                                                            | Required | Default value               |
```

Comment:
`instrument.sh` needs to be updated accordingly to the changes in the upsteam

Reply:
9efa13ff656a903e0fc6d509eccb3aa5935f5406